CondTools/Geometry/test/rpcgeometrywriter.py

```diff
@@ -4,6 +4,8 @@
 process.load('CondCore.CondDB.CondDB_cfi')
 process.load('Configuration.StandardSequences.GeometryExtended_cff')
 process.load('Geometry.MuonNumbering.muonNumberingInitialization_cfi')
+process.load("Geometry.MuonNumbering.muonGeometryConstants_cff")
+process.load('Configuration.StandardSequences.DD4hep_GeometrySim_cff')
```

Comment:
I don't think it is right to include this DD4hep geometry. This file needs to work with both old DD and DD4hep.

Reply:
@cvuosalo : I added the same line for PR #32001 (for CSC). Without it an error appears:

----- Begin Fatal Exception 12-Nov-2020 09:04:49 CET-----------------------
An exception of category 'NoProxyException' occurred while
   [0] Processing global begin Run run: 1
   [1] Calling method for module RPCRecoIdealDBLoader/'RPCGeometryWriter'
Exception Message:
No data of type "DDCompactView" with label "" in record "IdealGeometryRecord"
 Please add an ESSource or ESProducer to your job which can deliver this data.
----- End Fatal Exception -------------------------------------------------
%MSG-i DDCompactViewImpl:  12-Nov-2020 09:04:49 CET EndJob
DDD transient representation has been destructed.
%MSG